added meet a data steward events for the spring

annaasklof opened the pull request:
Added the "Meet a data steward" events for the spring. Dates can be changed in the future if the schedule is changed.

YvonneKallberg commented:
The file name (only, it is correct in the text) of the first event say 20230107.md, but should be 2023**02**07.md

In the update_event template, do we know that there are events, other than 'Meet a DS', where we want to use the template? If no, I would suggest to add the title, url and description in the template, so that the only change is the date. If yes, then don't change anything.

In the update_topic template:
* is the title supposed to always be 'New topic'? If no, then use <!-- --> instead
* the url, perhaps add the start for the url in the comment, to help us remember how it should be written?

annaasklof commented:
File name corrected. 

Good idea about the event template, I changed it to be only for "Meet a DS" events based on our discussion on the last meeting. 

For the topic templates, I added an example url. I thought the title always could be "New topic" but happy to change if we want something else?

YvonneKallberg commented:
Looks good, will do the merge
